spec/features/smoke_test_runner_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+feature 'Runner Smoke Test', type: :feature do
+  let(:smoke_test_form_url) do
+    ENV.fetch("SMOKE_TEST_FORM_URL") { raise "You must set $SMOKE_TEST_FORM_URL"}
+  end
+
+  before do
+    Capybara.app_host = smoke_test_form_url
+  end
+
+  scenario 'Complete and submit an existing form' do
+    logger.info "Visiting #{smoke_test_form_url}"
+    visit  smoke_test_form_url
+
+    expect(question_to_be_answered?).to be true
+
+    question_number = 1
+    while question_to_be_answered?
+      logger.info "Answering question: #{question_number}"
+      question_type = page.first('input')[:name]
+      fill_in(class: 'govuk-input', with: answer_for(question_type))
+      click_button 'Continue'
+      expect(page).to have_no_content('There is a problem')
```

Comment:
Curious on opinions on this please, might not be bullet-proof enough, e.g if we change the content etc...

Reply:
I've decided to check that after clicking 'continue' the page is not the same as it was before, e.g. it hasn't reloaded with some kind of validation error. If at any point we redirect to the Technical Errors page then that page will not have a question on it, and it won't have the check your answers text so the test will fail.